apps/Gruntfile.js

```diff
@@ -613,6 +613,7 @@ module.exports = function (grunt) {
 
   grunt.registerTask('prebuild', [
     'checkDropletSize',
+    'check-entry-points',
```

Comment:
Yeah - my impression is that this is more like a linting task than a build task, though we'd certainly like it to run as part of CI.  Maybe add this to linting and/or testing?